client/src/components/ProductListing/ProductListing.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+export default function ProductListing({
+  img,
+  brand,
+  model,
+  price,
+  sale = false,
+}) {
+  return (
+    <>
+      <ProductListElement>
+        <ProductImage src={img} />
+        <ProductInfo>
+          <ProductBrand>{brand}</ProductBrand>
+          <ProductModel>{model}</ProductModel>
+        </ProductInfo>
+        <ProductMeta>
+          <ProductPrice sale={sale}>
+            ab <span>{price} €</span>
+          </ProductPrice>
+        </ProductMeta>
+      </ProductListElement>
```

Comment:
To make this more readable, you could remove the prefix `Product` from all local styled components.

```suggestion
      <ListItem>
        <Image src={img} />
        <Info>
          <Brand>{brand}</Brand>
          <Model>{model}</Model>
        </Info>
        <Meta>
          <Price sale={sale}>
            ab <span>{price} €</span>
          </Price>
        </Meta>
      </ListItem>
```